udata_front/theme/gouvfr/datagouv-components/src/types/datasets.ts

```diff
@@ -37,6 +37,13 @@ export type BaseDataset = Owned & {
     zones?: Array<string>;
     granularity?: string;
   } | null;
+  contact_points: Array<{
```

Comment:
It seems that the API is sending :
- contact_form (can be null)
- email (can be null)
- id
- name
- role

Also, can you add this type as a separate one and use it in both datasets and dataservices files ?